lib/suse/connect/version.rb

```diff
@@ -1,5 +1,5 @@
 module SUSE
   module Connect
-    VERSION = '0.3.35'
+    VERSION = '0.4'
```

Comment:
According to the [semver docs](https://semver.org/#spec-item-2), I believe we'd need to make the version `0.4.0`

Reply:
That's part of the discussion we had with @ngetahun, in which for our products it doesn't make quite much sense to follow semver exactly. I believe Nate is to update the docs on our side, but the patch-level release number is no longer needed.

Reply:
Okay, thanks for the info! Seems I missed it